src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let callCount = 0;
+
+  return function(a) {
+    return function(b) {
+      callCount++;
+
+      if (callCount >= 4 && callCount % 2 === 0) {
+        return 'Bzzz... Error!';
+      }
 
+      const summOfInts = a + b;
```

Comment:
There is a typo in the variable name `summOfInts`. It should be `sumOfInts` to correctly represent 'sum of integers'.